app/src/organisms/FirmwareUpdateModal/FirmwareUpdateTakeover.tsx

```diff
@@ -3,43 +3,91 @@ import * as React from 'react'
 import {
   useInstrumentsQuery,
   useCurrentMaintenanceRun,
+  useCurrentAllSubsystemUpdatesQuery,
+  useSubsystemUpdateQuery,
 } from '@opentrons/react-api-client'
-
+import { Portal } from '../../App/portal'
 import { useIsUnboxingFlowOngoing } from '../RobotSettingsDashboard/NetworkSettings/hooks'
+import { UpdateInProgressModal } from './UpdateInProgressModal'
 import { UpdateNeededModal } from './UpdateNeededModal'
+import type { Subsystem } from '@opentrons/api-client'
 
-const INSTRUMENT_POLL_INTERVAL = 5000
+const POLL_INTERVAL_5000_MS = 5000
```

Comment:
If there isn't any specific reason to use 5000, removing it would be better since we basically don't include the number for this type of a const.

```suggestion
const POLL_INTERVAL_MS = 5000
```